### PR TITLE
Fix the MCP layer's presentation defects and release 2.0.0-beta.1

### PR DESCRIPTION
**Pull request by Anselmoo**
A session driving the `tanabesugano` MCP server by hand reported eleven defects. Eight are fixed and guarded, three were not defects against this tree, and one could not be settled — all recorded in `docs/mcp-defect-report-closure.md` so the report cannot be re-litigated from scratch.

What these defects had in common is that **none of them could fail a solver test**. The numbers were right in every case. What was wrong was which number got picked, which way an axis pointed, what a docstring promised, or what a result claimed about itself.

## Fixed

| | Commit |
| --- | --- |
| Density heatmap rendered its energy axis **inverted** — 0 cm⁻¹ at the top, 40,000 at the bottom | `ef653aa` |
| Ground term derived from the **Dq = 0 sample point**, where the whole manifold is exactly degenerate — misnamed d2, d6, d7 | `c86852c` |
| Dashboard sparkline spiked **26×–48×** at the origin, same cause | `c86852c` |
| A fitted Racah **C** was indistinguishable from a passthrough default | `51d024d` |
| `dq_max` units ambiguous against Δ = 10·Dq; ground-manifold exclusion undocumented | `51d024d` |
| No axis-limit parameters, so comparing two ions meant back-solving a sweep | `51d024d` |
| No selection-rule weighting on the landscape | `aaa1edc` |

## Not defects here

- **`GROUND_TERM_OCTAHEDRAL` wrong for d4/d6** — the symbol does not exist. Deleted as dead code; its successor already carries the values the report wanted.
- **`n_energy_points` undocumented** — it is documented, with its default.
- **PDF/SVG export rejected** — not this package. `ts_plot_png` already returns vector output as an `EmbeddedResource` rather than an image block, which *is* the remedy the report proposed. Anything still failing is the consuming client's handling of that content type.

## Corrections to the report

- The Dq = 0 tie-break was reported as affecting d3/d6/d8. Measured, it misnamed **d2/d6/d7** — d3 and d8 already resolved correctly.
- That row was rated a wrong *reported fact*. Its only output, `ground_y`, was discarded at all six call sites — latent, not user-visible, which is exactly why it outlived an earlier pass that fixed three sibling call sites. It was **deleted rather than corrected**, so no tempting local `min()` remains to copy.

## Worth a reviewer's attention

**Physics validates the sparkline fix independently.** After moving the sweep off the origin, d3/d4/d6/d8 report exactly 500/1000/1500 cm⁻¹ at Dq = 50/100/150 — ν₁ = 10Dq for the A₂g ground terms, and the 10Dq E–T₂ splitting for the D-term systems. Not asserted here: `test_ion_case_studies.py` already pins ν₁ = 10Dq, and a second copy at a second tolerance would let the looser mask the tighter.

**Selection-rule weighting is checked against group theory, not a fixture.** High-spin d5 holds exactly one sextet, so every d-d transition is forbidden and its column must scale by *exactly* the forbidden intensity. Measured: d5 → 0.050 exactly; d2/d8 agree to the digit, as do d4/d6 — hole conjugation, falling out unasked. A second test pins d3, because a blanket per-column factor would satisfy the d5 oracle and nothing else would notice.

**The Chart.js half is verified by execution, not by grep.** Carrying `y_min` in JSON and reading `p.y_min` in the widget are two claims across a language boundary neither language's tooling checks. `_render_scales()` strips the network import, stubs the sandbox, and runs the widget's own script under node against the real payload. It was mutation-tested before being kept: renaming the payload key left the source-grep guard **passing** and the execution guard failing.

**A second Chart.js renderer was found and deleted.** `_HEATMAP_HTML` carried the identical un-oriented axis, unreachable but one re-registration from restoring the bug. It was kept on the theory that a client referencing the URI would get "a clean not found rather than an import error" — measured, an unregistered URI and one that never existed both raise `NotFoundError: Unknown resource`, byte for byte. `test_every_chart_renderer_orients_its_energy_axis` is deliberately un-anchored so a third renderer cannot hide the same way.

**62 latent test failures fixed.** Five test files used a bare `asyncio.run()` on the main thread, which anyio's pytest plugin breaks. Only one ever failed, by pure alphabetical accident — `test_script_export` is the only module sorting after `test_screenshots`. Forcing the ordering the alphabet was hiding exposed 50 failures in `test_mcp.py` alone. CI never saw any of it, because `poe test` passes `--ignore=.../test_screenshots.py`.

## Release

`rrt bump beta` promoted `[Unreleased]` and moved `pyproject.toml`, `__init__.py` and `CITATION.cff` to **2.0.0-beta.1**.

## Verification

- `uv run poe check` clean — lint, format, `ty`, tests, artifact drift (`0 file(s) differ`).
- Whole suite green **without** the `--ignore` CI applies.
- Tree drift gate clean, lock refreshed 214 → 217.
- 50 new regression guards in `src/tanabesugano/test/test_mcp_regressions.py`, each recording its observed failure text.

## Open

The d8 fitter allegedly anchoring to a singlet ground term could not be settled: the original fixture was not found and this project's memory store is empty. It does not reproduce across seven cases including the spin-forbidden path, and `test_spectrum_fitting.py` already asserts the d8 ground term against the oracle table, so a singlet anchor would fail a standing test. Recorded as "cannot reproduce, and a guard would catch it" — not "was never real".

🤖 Generated with [Claude Code](https://claude.com/claude-code)

## Summary by Sourcery

Release 2.0.0-beta.1 with corrected MCP visualizations, clearer fit and plotting results, and regression coverage for presentation-layer defects.

New Features:
- Add optional selection-rule weighting to oxidation landscape visualizations and report whether spectrum fits constrain Racah C.
- Add configurable axis bounds to diagram and plot rendering tools.

Bug Fixes:
- Correct energy-axis orientation and bounds in Chart.js heatmaps.
- Prevent zero-field degeneracy from corrupting ground-term selection and dashboard first-excited-state sparklines.
- Clarify MCP behavior and parameter units, and remove the obsolete heatmap renderer.

Enhancements:
- Expose whether C is a default or physically constrained value in spectrum-fit results and warn when it is unconstrained.
- Consolidate event-loop-safe asynchronous test execution and add comprehensive MCP presentation regression coverage.

Build:
- Bump the package and release metadata to 2.0.0-beta.1 and refresh the lock/tree metadata.

Documentation:
- Document the closure status of the hand-driven MCP defect report.

Tests:
- Add regression tests covering rendering orientation, zero-field handling, selection-rule weighting, fit reporting, axis limits, documentation contracts, release metadata, and cross-language Chart.js behavior.

Chores:
- Remove dead heatmap resource code and synchronize citation release metadata.

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -7,7 +7,7 @@ authors:
 - family-names: "thebadgerchemist"
 - family-names: "Imgbot"
 title: "TanabeSugano"
-version: 2.0.0-alpha.2
+version: 2.0.0-beta.1
 doi: 10.5281/zenodo.7751230
 date-released: 2026-08-20
```
[the remark this answers is not in the document]
Comment (Anselmoo) [reply CITATION.cff]: Fixed in 2697398 — `date-released` is now 2026-08-23, matching the changelog. Root cause was systematic, not a typo: rrt rewrites `version:` in CITATION.cff via `[[tool.rrt.version_targets]]` but its schema has no date concept, so the field sat at the alpha.1 date through two bumps; added `test_release_metadata.py` pinning both version and date against `__version__` and the changelog so it cannot drift again.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_
